code/game/objects/structures/xeno.dm

```diff
@@ -169,11 +169,9 @@
 		playsound(src, "alien_resin_break", 25)
 		C.visible_message("<span class='warning'>[C] trips on [src]!</span>",\
 						"<span class='danger'>You trip on [src]!</span>")
-		C.Paralyze(40)
-		if(!QDELETED(linked_carrier) && linked_carrier.stat == CONSCIOUS && linked_carrier.z == z)
-			var/area/A = get_area(src)
-			if(A)
-				to_chat(linked_carrier, "<span class='xenoannounce'>You sense one of your traps at [A.name] has been triggered!</span>")
+		C.Paralyze(4 SECONDS)
+		if(linked_carrier)
+			xeno_message("<span class='xenoannounce'>A facehugger trap at [AREACOORD_NO_Z(src)] has been triggered!</span>", 2, linked_carrier.hivenumber,  FALSE, get_turf(src), 'sound/voice/alien_distantroar_3.ogg') //Follow the trend of hive wide alerts for important events
```

Comment:
use signals

Reply:
TBH will probably just tie this into the structure alert system once https://github.com/tgstation/TerraGov-Marine-Corps/pull/5335/files is merged

Reply:
this request is still standing

Reply:
5335 has this covered; why reinvent the wheel?